module-bsp/board/rt1051/bsp/torch/torch.cpp

```diff
@@ -42,11 +53,11 @@ namespace bsp
             gpio->ConfPin(DriverGPIOPinParams{.dir      = DriverGPIOPinParams::Direction::Output,
                                               .irqMode  = DriverGPIOPinParams::InterruptMode::NoIntmode,
                                               .defLogic = 0,
-                                              .pin      = static_cast<uint32_t>(BoardDefinitions::TORCH_DRIVER_EN)});
-            gpio->WritePin(static_cast<uint32_t>(BoardDefinitions::TORCH_DRIVER_EN), 0);
-            vTaskDelay(pdMS_TO_TICKS(5));
-            gpio->WritePin(static_cast<uint32_t>(BoardDefinitions::TORCH_DRIVER_EN), 1);
-            vTaskDelay(pdMS_TO_TICKS(5));
+                                              .pin = static_cast<std::uint32_t>(BoardDefinitions::TORCH_DRIVER_EN)});
+            powerOffTorch();
+            vTaskDelay(pdMS_TO_TICKS(delayTicks));
+            powerOnTorch();
+            vTaskDelay(pdMS_TO_TICKS(delayTicks));
```

Comment:
those are not ticks but milliseconds, so please rename the variable ;)

Reply:
You're right, thanks :)